docs/Users_Guide/working_with_templates.rst

```diff
@@ -156,15 +156,15 @@ The rendered template will be updated with the values contained in the config fi
 Input file and environment file
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-If templater.py is called on an input file but no config file is provided, the template will be rendered using the user environment.
+If ``templater.py`` is called on an input file but no config file is provided, the template will be rendered using the user environment.
 
 .. _temp_inp_cli:
 
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Input file and command line config items
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-templater.py can be run with an input file and config items provided through the command line by using the config_items flag::
+``templater.py`` can be run with an input file and config items provided through the command line by using the ``config_items`` flag::
 
     python src/uwtools/templater.py -i /<path-totemplate>/jinja2template.nml NFGRIDS=0, NMGRID=5, FUNIPNT=' T', IOSRV='None', FPNTPROC='None', FGRDPROC=' None'
```

Comment:
@christinaholtNOAA @fgabelmannjr Is `-i` the config_items flag? Also, why doesn't anything for `NMGRID=5` seem to appear in the rendered template below?

Reply:
templater.py has been updated since this draft of the User's Guide was written and it now handles missing values differently, so there are several sections of Working with Templates we need to rewrite (details are in #202 )

Reply:
@venitahagerty I have added the `-h` output (with a caveat that it may change), updated NMGRID to NMGRIDS, standardized the sample template name to `jinja2template.nml`, and removed the commas between the config items. It sounds like @j-derrico will take care of your last concern in a separate PR.